allas checks and updates (#2165)
CSCfi/csc-user-guide: docs/data/Allas/allas_project_example.md

Allas storage is only for research project's duration, but Saara thinks it would be beneficial to have the preliminary data made publicly available and easier to be found. This is supported by the [Fairdata Services](https://www.fairdata.fi/en/) produced by CSC.

> Pekka creates a new bucket with public access and uploads the data to the bucket. Command _a-publish_ creates such new bucket and uploads a files in to it. Parameter -b is used to define the name for the bucket, in this case _hiano-project-public001_.

Reviewer: ```suggestion
Pekka creates a new bucket with public access and uploads the data to the bucket. Command _a-publish_ creates the bucket and uploads the selected files to it. Parameter `-b` is used to define the name for the bucket, in this case `hiano-project-public001`.
```

```text
a-publish -b hiano-project-public001 zz_364872.dat zz_242165.dat
```

> Next Pekka describes basic information about the data in [Fairdata Qvain Tool](https://www.fairdata.fi/en/qvain/) and provides the two URLs (one for each file in Allas) as a Remote Resource in Qvain. After this the data can then be published as a dataset with a landing page and a persistent identifier. This way the preliminary data can be shared among colleagues using this persistent identifier. The dataset can also be explored via [Fairdata Etsin Service](https://www.fairdata.fi/en/etsin/) with structured information and direct access to download the files in Allas.

Reviewer: ```suggestion
Next, Pekka creates a basic description of the data using the [Fairdata Qvain Tool](https://www.fairdata.fi/en/qvain/) and provides the two URLs (one for each file in Allas) as a Remote Resource in Qvain. After this, the data can be published as a dataset with a landing page and a persistent identifier. This way the preliminary data can be shared among colleagues using the persistent identifier. The dataset can also be explored via [Fairdata Etsin Service](https://www.fairdata.fi/en/etsin/) with structured information and direct access to download the files in Allas.
```
